Src/Fido2/Fido2.cs

```diff
@@ -65,16 +65,18 @@ public CredentialCreateOptions RequestNewCredential(
     /// <param name="attestationResponse">The attestation response from the authenticator.</param>
     /// <param name="originalOptions">The original options that was sent to the client.</param>
     /// <param name="isCredentialIdUniqueToUser">The delegate used to validate that the CredentialID is unique to this user.</param>
+    /// <param name="requestTokenBindingId">DO NOT USE - Deprecated, but kept in code due to conformance testing tool</param>
     /// <param name="cancellationToken">The <see cref="CancellationToken"/> used to propagate notifications that the operation should be canceled.</param>
     /// <returns></returns>
     public async Task<RegisteredPublicKeyCredential> MakeNewCredentialAsync(
         AuthenticatorAttestationRawResponse attestationResponse,
         CredentialCreateOptions originalOptions,
         IsCredentialIdUniqueToUserAsyncDelegate isCredentialIdUniqueToUser,
+        byte[]? requestTokenBindingId = null,
```

Comment:
That's an interesting alternative to get it less visible!

Reply:
@iamcarbon While drafting an implementation on your feedback  I decided against it since it would pollute the CredentialCreateOptions; they are expected to get serialised and sent to the client and should contain the "request token id" value.